yarn.lock

```diff
@@ -11480,6 +11480,11 @@ prepend-http@^2.0.0:
   resolved "https://registry.yarnpkg.com/prepend-http/-/prepend-http-2.0.0.tgz#e92434bfa5ea8c19f41cdfd401d741a3c819d897"
   integrity sha1-6SQ0v6XqjBn0HN/UAddBo8gZ2Jc=
 
+prettier@^2.2.0:
+  version "2.2.0"
+  resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.2.0.tgz#8a03c7777883b29b37fb2c4348c66a78e980418b"
+  integrity sha512-yYerpkvseM4iKD/BXLYUkQV5aKt4tQPqaGW6EsZjzyu0r7sVZZNPJW4Y8MyKmicp6t42XUPcBVA+H6sB3gqndw==
+
```

Comment:
I don't see any references to it in any of this diff. Are you sure you are actually using it for your intended purpose?

Reply:
That was mentioned in the issue description, jest requires this dependency for inline snapshots: https://github.com/facebook/jest/issues/8467

Reply:
Ah I saw that comment, but I assumed that it would have to be explicitly "used" somewhere.